src/AutoCompleteTextField.js

```diff
@@ -457,15 +466,18 @@ class AutocompleteTextField extends React.Component {
           onClick={() => { this.handleSelection(idx); }}
           onMouseEnter={() => { this.setState({ selection: idx }); }}
         >
-          {val.slice(0, highlightStart)}
-          <strong>{val.substr(highlightStart, matchLength)}</strong>
-          {val.slice(highlightStart + matchLength)}
+
+          {renderOptionsList(<>{val.slice(0, highlightStart)}<strong>{val.substr(highlightStart, matchLength)}</strong>{val.slice(highlightStart + matchLength)}</>, val)}
         </li>
       );
     });
 
+    if (!expandUp) {
+      height = top + offsetY;
```

Comment:
The way logic is spreaded for height evaluation if confusing.

Something like below would be cleaner from my perspective:
```
const height = expandUp ? (offsetY + helperOptions.length * optionHeight) : top + offsetY;
```